src/app/faucet/page.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
```

Comment:
can we move the "use client" to the Faucet component instead of putitng it here?